crates/worker/src/partition/state_machine/effects.rs

```diff
@@ -416,24 +416,27 @@ impl Effect {
                 span_context,
                 ..
             } => match timer_value.value() {
-                Timer::CompleteSleepEntry(_) => {
+                Timer::CompleteSleepEntry(invocation_id, entry_index) => {
                     info_span_if_leader!(
                         is_leader,
                         span_context.is_sampled(),
                         span_context.as_parent(),
                         "sleep",
-                        restate.invocation.id = %timer_value.invocation_id(),
-                        restate.timer.key = %TimerKeyDisplay(timer_value.key()),
+                        restate.invocation.id = %invocation_id,
+                        restate.journal.index = entry_index,
                         restate.timer.wake_up_time = %timer_value.wake_up_time(),
+                        restate.timer.key = %TimerKeyDisplay(timer_value.key()),
                         // without converting to i64 this field will encode as a string
                         // however, overflowing i64 seems unlikely
                         restate.internal.end_time = i64::try_from(timer_value.wake_up_time().as_u64()).expect("wake up time should fit into i64"),
                     );
 
                     debug_if_leader!(
                         is_leader,
-                        restate.timer.key = %TimerKeyDisplay(timer_value.key()),
+                        restate.journal.index = entry_index,
+                        restate.invocation.id = %invocation_id,
```

Comment:
This is unnecessary repetition I think, you have the invocation id in the span already

Reply:
Ah ok, then I'll remove it. I was assuming it is not the case because the `info_span_if_leader` was containing it.

Reply:
A lot of other effects still have the `restate.invocation.id` as an event attribute. What is the guiding principle here?

Reply:
When there is a caller and callee, you put the callee invocation id too in the event itself and not the span.

Reply:
> When there is a caller and callee, you put the callee invocation id too in the event itself and not the span.

I assume that for `SetState` there is no callee, right? It seems that we are still logging `restate.invocation.id`. Could it be that we didn't properly clean it up after the latest changes to the how the span is initialized?